package.json

```diff
@@ -3,7 +3,6 @@
   "private": true,
   "version": "0.0.0",
   "license": "Apache-2.0",
-  "type": "module",
```

Comment:
Why remove `type`

Reply:
Due to issues in Prettier with ESModules, it cannot used with `.js` even though I tried `v3`. But in our case we don't need `.js` actually, so reverted back to `module` and upgraded Prettier now